packages/sdk/src/types/Consents.ts

```diff
@@ -0,0 +1,30 @@
+export interface ConsentsListResponse {
+	groups?: {
+		object?: 'list'
+		total?: number
+		data_ref?: 'data'
+		data: {
+			id?: string
+			name?: string
+			description?: string
+			consents?: string[]
+			created_at?: string
+			updated_at?: string
+			object?: 'consent_group'
+		}[]
+	}
+	consents?: {
+		object?: 'list'
+		total?: number
+		data_ref?: 'data'
+		data?: {
```

Comment:
Please create similar small type for this like for `interface ConstentGroup` and use type array here.